testdata/ing_shared_rg_notshared_alb/input/k8s/01-ing.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: myingress
+  annotations:
+    zalando.org/aws-load-balancer-shared: "true"
```

Comment:
test name is called "ing_shared_.." which suggests that it should be `shared: false`

Reply:
The ingress is shared, the RG not shared. At least that was my intent with the naming. :S

The idea of this test is to emulate a scenario where we have a ingress that shares an LB with other ingresses and RouteGroups and a RouteGroup with a special case that does not wish to share the LB. That is why one of them is `true` and the other is `false`.

I know it can be confusing, but I also don't have better ideas for the name of this test. Do you have any suggestions?

Reply:
So basically your test run uses one ingress and one routegroup and compares what happens if you have on "not shared" and the other "shared". Ok, now it makes sense to me.